messages/autoware_external_api_msgs/msg/ClassifiedDiagnostics.msg

```diff
@@ -0,0 +1,5 @@
+builtin_interfaces/Time stamp
+# TODO: move diagnostics
+# TODO: move diagnostics_error
+diagnostic_msgs/DiagnosticStatus[] diagnostics_notice
+# TODO: move hazard_status
```

Comment:
Could you remove all TODO?

Reply:
OK !
Thanks for your review.
fix https://github.com/autowarefoundation/autoware.universe/pull/20/commits/e8218d792733898f3bb48403bacd59b2983fb99a